functional/FunctionalTest.java

```diff
@@ -251,22 +251,21 @@ public static void makeBucket_test1() throws Exception {
     }
   }
 
-  /** Test: makeBucket(String bucketName, String region). */
+  /** Test: makeBucket(MakeBucketArgs args). */
   public static void makeBucketwithRegion_test() throws Exception {
     if (!mintEnv) {
-      System.out.println("Test: makeBucket(String bucketName, String region)");
+      System.out.println("Test: makeBucket(MakeBucketArgs args)");
```

Comment:
You can add a tag like `with region`, `with objectLock` or `with region and objectLock`